주민_강민주/3주차_미션/미션3/src/pages/Movies.tsx

```diff
@@ -0,0 +1,66 @@
+ import {useState} from "react"
+import MovieCard from "../components/movieCard";
+import { useParams } from "react-router-dom";
+import useFetchMovies from "../hooks/uesFetchMovies";
+
+const Movies=()=>{
+    const {category}=useParams();
+    const [page, setpage]=useState(1);
+    const {data, loading, error}=useFetchMovies(category,page);
+
```

Comment:
```suggestion
    useEffect(() => {
        setpage(1);
    }, [category]);
```

지금은 예를 들어 인기영화에서 3페이지 영화를 불러왔다가, 상영중으로 넘어가도 똑같이 3페이지 데이터가 불러와집니다! category를 의존성 배열에 추가하여 category가 변경될 때 page를 1로 초기화 해주면 이러한 문제를 해결 할 수 있습니다!